Apps/frontend/src/OscilloscopeWebGl.ts

```diff
@@ -100,6 +105,31 @@ export class OscilloscopeWebGl {
       this.webgl.uniform1f(amplitudeUniform, channelAmplitude);
 
       this.webgl.drawArrays(this.webgl.LINE_STRIP, 0, samples.length);
+
+      // draw twice to make thicker lines
+      if (get(thicknessAdjustment)[i]) {
+        for (let j = -LINE_THICKNESS_DUPLICATES; j < 0; j++) {
+          offsetUniform = this.webgl.getUniformLocation(
+            this.channelProgram,
+            "u_offset"
+          );
+          channelOffset = get(offsetAdjustment)[i] + j * LINE_THICKNESS_DELTA;
```

Comment:
Would it be possible to move this operation into/compute the second line in the vertex shader?
Don't think we'll run into any major performance problem anyway, but IMO it would be better suited in there.